### PR TITLE
Update documentation about Qiskit Transpiler Service & AI transpiler passes

### PR DESCRIPTION
**Pull request by cbjuan**
Adding documentation of the new features and changes available after the release of versions 0.2.0 & 0.2.1 of the `qiskit_transpiler_service`

### CODE REVIEW

File: docs/transpile/ai-transpiler-passes.mdx [hunk 1]
```diff
@@ -11,15 +11,14 @@ The AI-powered transpiler passes are experimental passes that work as a drop-in
 <Admonition type="note">
     This is an experimental feature available only to the IBM Quantum Premium Plan.
     The AI-powered transpiler passes are in alpha release status, subject to change.
+    If you have feedback or want to contact the developers team, please use this [Qiskit Slack Workspace channel](https://qiskit.slack.com/archives/C06KF8YHUAU)
```
Comment (abbycross) [inline docs/transpile/ai-transpiler-passes.mdx:14]:
```suggestion
    If you have feedback or want to contact the developer team, please use this [Qiskit Slack Workspace channel](https://qiskit.slack.com/archives/C06KF8YHUAU).
```

File: docs/transpile/qiskit-transpiler-service.mdx [hunk 2]
```diff
@@ -12,6 +12,7 @@ The Qiskit transpiler service offers a Python library to seamlessly integrate th
 <Admonition type="note">
     This experimental service is only available for IBM Quantum Premium Plan users.
     The service is an alpha release, subject to change.
+    If you have feedback or want to contact the developers team, please use this [Qiskit Slack Workspace channel](https://qiskit.slack.com/archives/C06KF8YHUAU)
```
Comment (abbycross) [inline docs/transpile/qiskit-transpiler-service.mdx:15]:
```suggestion
    If you have feedback or want to contact the developer team, please use this [Qiskit Slack Workspace channel](https://qiskit.slack.com/archives/C06KF8YHUAU).
```

**Review by abbycross — APPROVED** [review]
I made a few wordsmithing/grammatical tweaks. 
Looks good to me 👍

**Comment by cbjuan** [comment]
Great, thanks @abbycross! I just made one last commit with a minor update to the code, to improve the coherence of all the examples

**Pull request merged**